data/migrations/V0083__fix_ofec_sched_a_aggregate_state_recipient_totals_mv.sql

```diff
@@ -0,0 +1,175 @@
+-- Superceeds Migration #54
+-- redefine MATERIALIZED VIEW public.ofec_sched_a_aggregate_state_recipient_totals_mv to join properly to staging.ref_zip_to_district
+
+REFRESH MATERIALIZED VIEW public.ofec_committee_history_mv;
+REFRESH MATERIALIZED VIEW public.ofec_committee_detail_mv;
+
```

Comment:
Tested locally and it looks good.
Line 4 and line 5 may not be necessary since these two materialized views are refreshed every day.